gateway/1.4/modules/ROOT/pages/flex-install.adoc

```diff
@@ -203,14 +199,14 @@ To install Flex Gateway in a Docker container:
 docker pull mulesoft/flex-gateway:{gateway-ver-var}
 ----
 
-. Verify the installation was successful: 
+. Verify that the download succeeded: 
 +
 [source,ssh]
 ----
 docker images
 ----
 +
-You should see the Flex Gateway container image in the resulting list. 
+When successful, the command lists the Flex Gateway container image, for example: 
```

Comment:
I suggest changing the command `docker images` to `docker images mulesoft/flex-gateway` to list only flex images

Reply:
Also, it might be valuable to extract the image name `mulesoft/flex-gateway` to a var/constant. It's unlikely that it changes, but just in case

Reply:
done and tested